.github/workflows/python.yml

```diff
@@ -85,16 +90,22 @@ jobs:
         shell: bash
 
       - name: run pytest
-        run: python -m pytest
+        run: python -m pytest --cov
+
+      - name: codecov
+        uses: codecov/codecov-action@v3
```

Comment:
This will only upload the results executed from `pytest --cov` above, right?

Reply:
Yes

Reply:
Sounds good. C# side would be neat for the further future when this is a bit more out of alpha and when i'm actually writing tests for it.